src/fs_core.ts

```diff
@@ -37,10 +39,13 @@ export interface FileSystemSyncAccessHandle {
 }
 
 export class SyncOPFSFile {
+  // FIXME needs a close() method to be called after start() to release the underlying handle
   constructor(public handle: FileSystemSyncAccessHandle) { }
 
-  open() {
-    return new OpenSyncOPFSFile(this);
+  open(fd_flags: number) {
```

Comment:
@bjorn3 Added handling for `FDFLAGS_APPEND` here and in `File` to be able to correctly *append* to existing files.